ktlint-baseline.xml

```diff
@@ -89,44 +89,6 @@
         <error line="803" column="9" source="spacing-between-declarations-with-comments" />
         <error line="849" column="89" source="trailing-comma-on-call-site" />
     </file>
-    <file name="components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt">
-        <error line="90" column="37" source="trailing-comma-on-declaration-site" />
-        <error line="107" column="5" source="spacing-between-declarations-with-comments" />
-        <error line="149" column="52" source="trailing-comma-on-declaration-site" />
-        <error line="151" column="5" source="spacing-between-declarations-with-annotations" />
-        <error line="161" column="48" source="trailing-comma-on-declaration-site" />
-        <error line="192" column="28" source="trailing-comma-on-declaration-site" />
```

Comment:
Should this be part of this commit? :)

Reply:
Had to delete and regenerate this baseline since it was having 100 of issues that were not caused by my changes.

Reply:
There is still some [tasks](https://github.com/mozilla-mobile/android-components/pull/12933/checks?check_run_id=8921452628) failing, could you try not checking this file to see if still fails in CI?